packages/connect/README.md

```diff
@@ -43,6 +43,17 @@ Since version 9 we are adopting a new versioning strategy. With every release, w
 
 Version 9 is available as `@trezor/connect` and `@trezor/connect-web` npm packages.
 
+### Major version
+New major version is released when there are breaking changes on API level. Previous major version will be mantained for 12 months and after this period it can be taken down without notice
```

Comment:
maybe constion `WHEN` and  information about `HOW LONG` should be separated? I could imagine also a table with exact dates when major version releases happend and how long they will be receiving security updates and when they were finally killed completely?